test_sets/avoid_custom_subscription_and_capabilities.txt

```diff
@@ -0,0 +1,11 @@
+./test_scripts/API/ButtonSubscription/Capabilities/001_SubscribeButton_CUSTOM_BUTTON_missing_in_hmiCap_file_and_supported_by_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/002_SubscribeButton_CUSTOM_BUTTON_missing_in_hmiCap_file_and_not_supported_by_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/003_SubscribeButton_CUSTOM_BUTTON_exists_in_hmiCap_file_and_not_supported_by_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/004_SubscribeButton_CUSTOM_BUTTON_missing_in_hmiCap_file_and_invalid_from_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/005_SubscribeButton_CUSTOM_BUTTON_exists_in_hmiCap_file_and_invalid_from_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/006_SubscribeButton_Unsubscribe_CUSTOM_BUTTON_UNSUPPORTED_RESOURCE.lua
+./test_scripts/API/ButtonSubscription/Capabilities/007_SubscribeButton_Unsubscribe_CUSTOM_BUTTON_UNSUPPORTED_RESOURCE_multiple_apps.lua
+./test_scripts/API/ButtonSubscription/Capabilities/008_SubscribeButton_CUSTOM_BUTTON_resumption_Unexpected_disconnect_not_supported_by_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/009_SubscribeButton_CUSTOM_BUTTON_resumption_IGN_OFF_ON_not_supported_by_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/010_SubscribeButton_CUSTOM_BUTTON_resumption_Unexpected_disconnect_supported_by_HMI.lua
+./test_scripts/API/ButtonSubscription/Capabilities/011_SubscribeButton_CUSTOM_BUTTON_resumption_IGN_OFF_ON_supported_by_HMI.lua
```

Comment:
What about scenarios:
1. When Button becomes supported by HMI after Ignition cycle (e.g. HMI was upgraded)
2. When Button becomes not supported by HMI after Ignition cycle (e.g. HMI was downgraded)

?